smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/IndexGenerator.java

```diff
@@ -82,6 +82,9 @@ static void writeServerIndex(
         writer.write("export * from \"./operations\";");
 
         writer.write("export * from \"./$L\"", symbol.getName());
+
+        writer.write("export { SmithyException } from $S", TypeScriptDependency.SERVER_COMMON.packageName);
```

Comment:
Maybe there's a reason I'm not aware of, but why is this here v/s AddBaseExceptionClassExport for the re-export on client side? If they can be same, seems like having it here is better v/s a new TypeScriptIntegration.

Reply:
The `writeServerIndex()` doesn't call the customer TypeScriptIntegration. Generally the TypeScriptIntegration interface is not fully supported in SSDK. So I leave them here. But I can apply the `writeAdditionalExports()` of `TypeScriptIntegration` for SSDK if you want.